### ISSUE TITLE
percentages gerapporteerd afschot per regio en per leeftijdscategorie inconsequent

### ISSUE THREAD

**Issue by SanderDevisscher** [inbo/reporting-rshiny-grofwildjacht#250]
# Title Figuur/Tabel: TABEL: Gerapporteerd afschot per regio en per leeftijdscategorie
## Bug description:
Wanneer je de tabel doorheen de tijd raadpleegt verwacht je dat er zodra er zowel data is voor jaar x als jaar x - 1|x - 5|x - 10 dat de verandering in percent weergegeven wordt. Maar dit is niet het geval, neem bvb voeren 

in de tabel van 2021 (update november branch lokaal in docker) wordt enkel de verandering t.o.v. 2016 (x-5) weergegeven, zie onder: 
![image](https://user-images.githubusercontent.com/16779320/141487860-03d423d6-2b11-4f84-b142-fa544c74310b.png)

in prd wordt de vergelijking t.o.v. 2016 niet weergegeven:
![image](https://user-images.githubusercontent.com/16779320/141488890-68072ea7-ddf7-4e74-9749-8dc89b030e96.png)

terwijl er zeker ook afschot in voeren is voor 2020 (x-1) & 2011 (x-10)

tabel 2020:
![image](https://user-images.githubusercontent.com/16779320/141488169-be9a3b10-904d-430a-9a07-93e442b9ca11.png)

tabel 2011:
![image](https://user-images.githubusercontent.com/16779320/141488257-c163f665-0692-4075-b1c8-e6a1c228382f.png)

**Comment by mvarewyck** [comment]
This is due to the fact that we only report percentages if there are more than 50 observations, see e.g. https://github.com/inbo/reporting-rshiny-grofwildjacht/issues/266#issuecomment-1012909374
The 50 observations rule applies both to the current year AND x-1 | x-5 | x-10 years before. So we report the percentage for x-5 only if the current and x-5 years have more than 50 observations.

This can indeed be confusing. We could also make this minimum a user input or have an alternative representation in the table e.g. percentage still printed but in red with warning in the description text.

**Comment by SanderDevisscher** [comment]
> This is due to the fact that we only report percentages if there are more than 50 observations, see e.g. [#266 (comment)](https://github.com/inbo/reporting-rshiny-grofwildjacht/issues/266#issuecomment-1012909374) The 50 observations rule applies both to the current year AND x-1 | x-5 | x-10 years before. So we report the percentage for x-5 only if the current and x-5 years have more than 50 observations.

ifso then there is something wrong allready since in my example only the current year has a total > 50 observations while the x-5 (2016) has not. Still the trend is displayed.  

> This can indeed be confusing. We could also make this minimum a user input or have an alternative representation in the table e.g. percentage still printed but in red with warning in the description text.

I would argue in favor of the following:
1. We maintain the 50 obs - rule, but
2. We color the trend yellow when one of the years has < 50 obs (like example)
3. and we color the trend red when both of the years have < 50 obs

we also provide a warning saying something like this: "LET OP! Wanneer de trend geel of rood gekleurd is de trend gebaseerd op minder dan 50 geschoten dieren en kan de geobserveerde trend eerder een gevolg zijn van toeval"

@jimcasaer what do you think ?

**Comment by mvarewyck** [comment]
> ifso then there is something wrong allready since in my example only the current year has a total > 50 observations while the x-5 (2016) has not. Still the trend is displayed.

I can't reproduce this. With the online app I have 130 observations for wild zwijn in 2016 and indeed only 50 for 2020.

![Screenshot from 2022-04-26 11-18-30](https://user-images.githubusercontent.com/22143743/165267446-5739208e-a220-4896-9900-657d1ad7e488.png)

**Comment by mvarewyck** [comment]
I've implemented a proposal. 

Remarks:

- If either the current or (1/5/10) previous year has 0 observations, we still don't show the trend. Otherwise this would result in -Inf, +Inf labels in the table
- We changed to code to have color if < min. for trend (50) instead of <= min. for trend (50) in accordance with the warning text
- Same changes apply to WBE table for Species
- Downloaded table has extra columns e.g. 'Warning 1 jaar' with zwart, oranje, rood indicating how to color the trend numbers (yellow is not visible). 

Example table Grofwild:
![Screenshot from 2022-04-26 12-24-26](https://user-images.githubusercontent.com/22143743/165279842-db40ba6b-b9ae-462a-9938-d4221ec93a25.png)

Example table WBE:
![Screenshot from 2022-04-26 12-44-33](https://user-images.githubusercontent.com/22143743/165283299-b8eb3a25-175d-4013-9dc1-1d219ab15b18.png)
